packages/reports/addon/templates/components/common-actions/schedule.hbs

```diff
@@ -129,13 +97,11 @@
                 {{else}}
                   Close
                 {{/if}}
-              </button>
+              </DenaliButton>
             {{/if}}
-          </div>
-          <div class="button-group">
             {{#unless this.localDeliveryRule.isNew}}
               <CommonActions::Delete
```

Comment:
This doesn't show up like a button anymore

Reply:
I'm aware of this, the issue is that updating the styles of this button effects all of the other buttons that spawns the delete modal, and I didnt know if I should change that yet considering the others havent been changed per Jon's request.